features/daily/directory.feature

```diff
@@ -0,0 +1,11 @@
+Feature: Directory integration
+
+    Scenario: List conference rooms as a dird user
+        Given there is a user
```

Comment:
Should we precise that the user is an authentication user and not an telephony user?
`Given there is an authentication user`